affiliates/estonia.swagger.json

```diff
@@ -0,0 +1,515 @@
+          "required" : false,
+          "style" : "simple",
+          "explode" : false,
+          "schema" : {
+            "format" : "string"
+          },
+          "example" : "et-EE"
+        }, {
+          "name" : "customer",
+          "in" : "query",
+          "description" : "Customer uuid from `/customers` response",
+          "required" : true,
+          "style" : "form",
+          "explode" : true,
+          "schema" : {
+            "format" : "string"
+          },
+          "example" : "df0f997b-72ae-480a-88a8-4bd73ee89dae"
+        } ],
+        "requestBody" : {
+          "description" : "Lead info",
+          "content" : {
+            "application/json" : {
+              "schema" : {
+                "$ref" : "#/components/schemas/Lead"
+              }
+            },
+            "application/xml" : {
+              "schema" : {
+                "$ref" : "#/components/schemas/Lead"
+              }
+            }
+          }
+        },
+        "responses" : {
+          "200" : {
+            "description" : "Success",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/LeadResponse"
+                }
+              },
+              "application/xml" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/LeadResponse"
+                }
+              }
+            }
+          },
+          "401" : {
+            "description" : "Unauthorized",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/UnauthorizedResponse"
+                }
+              },
+              "application/xml" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/UnauthorizedResponse"
+                }
+              }
+            }
+          },
+          "422" : {
+            "description" : "Validation error",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/ValidationErrors"
+                }
+              },
+              "application/xml" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/ValidationErrors"
+                }
+              }
+            }
+          }
+        },
+        "security" : [ {
+          "oAuth" : [ "allow_custom_pingtree" ]
+        } ]
+      }
+    }
+  },
+  "components" : {
+    "securitySchemes": {
+      "oAuth": {
+        "type": "oauth2",
+        "flows": {
+          "clientCredentials": {
+            "tokenUrl": "/token",
+            "scopes": { 
+              "allow_custom_pingtree": ""
+            }
+          }
+        }
+      },
+      "bearerAuth": {
+        "type": "http",
+        "scheme": "bearer"
+      }
+    },
+    "schemas" : {
+      "EmployerAddress" : {
```

Comment:
I'd describe employer_address as a whole better since as I understand its not employer address, its your own company's address.

Reply:
Added description: "Your company address"